packages/@sanity/field/src/diff/components/DiffErrorBoundary.tsx

```diff
@@ -39,17 +39,25 @@ export class DiffErrorBoundary extends React.Component<BoundaryProps, BoundarySt
 
     const help = isDev ? <p>Check the developer console for more information.</p> : null
     return (
-      <>
-        <div className={styles.root}>
-          <span className={styles.icon}>
-            <ErrorOutlineIcon />
-          </span>
-          <div className={styles.message}>
-            The component responsible for showing the changes to this field has crashed.
-            {help}
-          </div>
-        </div>
-      </>
+      <Card tone="critical" padding={3} paddingBottom={0}>
+        <Flex align="flex-start">
+          <Box>
+            <Text>
+              <Icon symbol="error-outline" />
+            </Text>
+          </Box>
+          <Box paddingLeft={3}>
+            <Text size={1} as="p">
+              The component responsible for showing the changes to this field has crashed.
+            </Text>
+            {help && (
+              <Text size={1} as="p">
```

Comment:
Nitpick: I _think_ it would look good to use `muted` on this `Text`. So the headline is a little darker than the copy below it.

Reply:
Good thinking!

Reply:
Looks kind of funny 🤔 ![Screenshot 2021-06-10 at 13 38 09](https://user-images.githubusercontent.com/25737281/121518722-19146380-c9f1-11eb-81f0-86346ff4bc47.png)

Reply:
Agreed, but let’s fix the theme (in another PR, as mentioned in catch-up)